lib/SILOptimizer/IPO/UsePrespecialized.cpp

```diff
@@ -88,20 +88,19 @@ bool UsePrespecialized::replaceByPrespecialized(SILFunction &F) {
     if (Subs.empty())
       continue;
 
-    ReabstractionInfo ReInfo(ReferencedF, Subs);
-
-    auto SpecType = ReInfo.getSpecializedType();
-    if (!SpecType)
+    // Bail if any generic type parameters are unbound.
+    if (hasUnboundGenericTypes(Subs))
```

Comment:
The actual limitation is/was that we only supported pre-specializations for full-specializations so far. There are no principal issues. But I want to wire this up once we have landed all partial specialization related changes. To work properly this code depends on proper mangling, proper visibility, etc.